website/docs/getting-started/setup/installation-guides/kubernetes/README.mdx

```diff
@@ -0,0 +1,353 @@
+| `persistence.enabled`               | Enable persistence using Persistent Volume Claims                    | `true`             |
+| `persistence.storageClass`          | Persistent Volume storage class                                      | `""`               |
+| `persistence.annotations`           | Additional custom annotations for the PVC                            | `{}`               |
+| `persistence.localStorage`          | Enable persistent volume using local storage                         | `false`            |
+| `persistence.storagePath`           | Local storage Path                                                   | `/tmp/hostpath_pv` |
+| `persistence.localCluster`          | Local running cluster to provide storage space                       | `[minikube]`       |
+| `persistence.accessModes`           | Persistent Volume access modes                                       | `[ReadWriteOnce]`  |
+| `persistence.size`                  | Persistent Volume size                                               | `10Gi`             |
+| `storageClass.enabled`              | Enable Storage Class configuration                                   | `false`            |
+| `storageClass.defaultClass`         | Create default Storage Class                                         | `false`            |
+| `storageClass.bindingMode`          | Binding mode for Persistent Volume Claims using Storage Class        | `Immediate`        |
+| `storageClass.allowVolumeExpansion` | Allow expansion of Persistent Volume Claims using Storage Class      | `true`             |
+| `storageClass.reclaimPolicy`        | Configure the retention of the dynamically created Persistent Volume | `Delete`           |
+| `storageClass.provisioner`          | Storage Class provisioner                                            | `""`               |
+| `storageClass.annotations`          | Additional storage class annotations                                 | `{}`               |
+| `storageClass.mountOptions`         | Mount options used by Persistent Volumes                             | `{}`               |
+| `storageClass.parameters`           | Storage Class parameters                                             | `{}`               |
+
+#### Auto update image
+
+| Name                   | Description                                   | Value         |
+| ---------------------- | --------------------------------------------- | ------------- |
+| `autoupdate.enabled`   | Enable auto update Helm chart's image         | `true`        |
+| `autoupdate.scheduler` | Schedule time to run cron job to update image | `"0 * * * *"` |
+
+When using command, specify each parameter using `--set key=value[,key=value]` argument to helm install. For example, the below command deploys Appsmith and configures it to use the storage class name `appsmith-pv`. 
```

Comment:
I now see why you had a separate sub-heading for this. I guess this is talking about the two ways users can specify parameters, isn't it? So, instead of the earlier 'Configure parameters' subheading (which I earlier asked to remove as it looked redundant), can we make the heading any one of these - Define parameters / Specify parameters / Set parameters, whichever makes most sense

Reply:
Set parameters - seems a good option to me. I'll add that.

Reply:
Done.